apps/dashboard/app/apps/ood_app_group.rb

```diff
@@ -42,9 +42,9 @@ def nav_limit
   # specified by 'group_by', sorting both groups and apps arrays by title
   def self.groups_for(apps: [], group_by: :category, nav_limit: nil)
     apps.group_by { |app|
-      app.send(group_by)
+      app.try(group_by)
     }.map { |k,v|
-      OodAppGroup.new(title: k, apps: v.sort_by { |a| a.title }, nav_limit: nav_limit)
+      OodAppGroup.new(title: k || I18n.t('dashboard.not_grouped'), apps: v.sort_by { |a| a.title }, nav_limit: nav_limit)
```

Comment:
I think this method may be used to build the menus too. The "not grouped" category should omit a title instead of having the title "Not Grouped". So this probably should be moved to the pinned_apps view (if title is nil, display the `I18n.t('dashboard.not_grouped')`.

Reply:
In the next PR related to metadata, I was going to force the order here. So that the "not grouped" group is always last. Is that something we want to consider?

Reply:
This is completed and moved to the view.